topiary/commandline_args.py

```diff
@@ -28,6 +28,7 @@
 from .parsing_helpers import parse_int_list
 from .rna import (
     load_cufflinks_fpkm_dict,
```

Comment:
Yeah, that was my understanding: but why use `tracking` in some places and `cufflinks` in other places?